applications/nrf_desktop/doc/ble_adv.rst

```diff
@@ -3,4 +3,56 @@
+| :ref:`ble_bond`  | ``ble_peer_operation_event`` |             |                           |                   |
++------------------+------------------------------+             +---------------------------+-------------------+
+|                  |                              |             | ``ble_peer_search_event`` | :ref:`led_state`  |
++------------------+------------------------------+-------------+---------------------------+-------------------+
+
+
+Configuration
+*************
+
+The module requires the basic Bluetooth configuration, as described in the Bluetooth guide.
+
+Set the ``CONFIG_DESKTOP_BLE_ADVERTISING_ENABLE`` to enable the BLE advertising module for the nRF Desktop peripheral device.
+
+You must define the short device name used for advertising (``CONFIG_DESKTOP_BLE_SHORT_NAME``), the Bluetooth device name (:option:`CONFIG_BT_DEVICE_NAME`) and the Bluetooth device appearance (:option:`CONFIG_BT_DEVICE_APPEARANCE`).
+
+By default, the module uses indirect advertising.
+Set the ``CONFIG_DESKTOP_BLE_DIRECT_ADV`` option to use direct advertising.
+
+.. tip::
+   The nRF Desktop peripheral will not advertise directly towards central that uses Resolvable Private Address.
+   The peripheral does not check if the remote device supports address resolution of directed advertisements - it does not read the Central Address Resolution GATT characteristic of the central.
+
+Set the ``CONFIG_DESKTOP_BLE_FAST_ADV`` to make the peripheral initially advertise with shorter interval to enquicken finding the peripheral by Bluetooth centrals.
+If the device uses indirect advertising, it will switch to slower advertising after the ``CONFIG_DESKTOP_BLE_FAST_ADV_TIMEOUT`` seconds.
+If the device uses direct advertising, you will observe the ``ble_peer_event`` with :cpp:member:`state` set to :cpp:enumerator:`PEER_STATE_CONN_FAILED` in case the Bluetooth central does not connect during predefined period of fast direct advertising.
+After the event is received, the device will switch to low duty direct advertising.
+Switching to slower advertising is done to reduce the energy consumption.
+
+You can use the ``CONFIG_DESKTOP_BLE_SWIFT_PAIR`` option to enable the Swift Pair feature.
+The feature simplifies pairing the Bluetooth peripheral to Windows10 hosts.
+
+When system goes to power down state, the advertising is stopped.
+If the Swift Pair was enabled, the device advertises without the Swift Pair data for additional ``CONFIG_DESKTOP_BLE_SWIFT_PAIR_GRACE_PERIOD`` seconds to ensure that user will not try to connect to the device that is no longer available.
+
+Implementation details
+**********************
+
+If the currently used Bluetooth local identity already has a bond and the device uses indirect advertising, it will not set General Discoverable flag.
+If the :option:`CONFIG_BT_WHITELIST` is enabled, the device will also whitelist incoming scan response data requests and connection requests.
```

Comment:
Is this related to the flag mentioned in the sentence above?

Reply:
It has similar purpose, but these are two separate actions.